lms/static/scripts/postmessage_json_rpc/client.js

```diff
@@ -36,7 +31,7 @@ async function call(
   params = [],
   timeout = 2000,
   window_ = window,
-  id = generateId()
+  id = generateHexString(10)
```

Comment:
The `generateId` helper wasn't needed. It's just a trivial function call and the usage is obvious from the context.